pkg/ocgorm/stats.go

```diff
@@ -168,6 +177,14 @@ func RecordStats(db *gorm.DB, interval time.Duration) (fnStop func()) {
 			select {
 			case <-ticker.C:
 				dbStats := db.DB().Stats()
+
+				if dbStats.OpenConnections == 0 { // We cleanup the ticker in the event that the database is unavailable
```

Comment:
Wish this had a test that exercised this logic, but sadly this project has very few tests.

Reply:
Yeah with tighter ownership of these packages I'd be happy to - at the moment I'm just extending what's in place. Cloud-sdk and it's sister packages REALLY need a charter and concrete ownership.